mhs-reference-implementation/mhs/SDSLookup/tests/test_sds_handler.py

```diff
@@ -0,0 +1,36 @@
+from unittest import TestCase
+import mhs.SDSLookup.sds_handler as sds
+
+import mhs.SDSLookup.tests.ldap_mocks as mocks
+from utilities.test_utilities import async_test
+
+NHS_SERVICES_BASE = "ou=services, o=nhs"
+
+MHS_OBJECT_CLASS = "nhsMhs"
+
+OPENTEST_SDS_URL = "192.168.128.11"
+PARTY_KEY = " A91461-9199084"
+SERVICE_ID = "urn:nhs:names:services:psis:MCCI_IN010000UK13"
+ORG_CODE = " A91461"
+
+
+class TestSDSHandler(TestCase):
```

Comment:
I'll change the the test to match the class name since I think we discussed previously that routing/reliability probably doesn't need to know that the information comes from SDS, just that they're attributes of an MHS. Does that still seem reasonable?

Reply:
Yes, that's a fair point, I'm happy to go with that.